src/modules/reporting/components/reporting-report/reporting-report.jsx

```diff
@@ -88,8 +92,21 @@ export default class ReportingReport extends Component {
         reporterGasCost: formatEtherEstimate(marketCreationCostBreakdown.targetReporterGasCosts),
         stake: this.props.isOpenReporting ? '0' : repAmount.formatted,
       })
+    })
+  }
+
+  calculateGasEstimates() {
```

Comment:
Please breakout out into a util file and add test coverage.

Reply:
ok, will break out.